src/en/news/blog/2022/improving-dashboard-accessibility-for-blind-and-visually-impaired-ceph-users/index.md

```diff
@@ -0,0 +1,128 @@
+speed. When sites are correctly designed, developed and edited, more users have equal access
+to information and functionality.([Wikipedia](https://en.wikipedia.org/wiki/Web_accessibility))
+
+The Ceph Dashboard is a built-in web-based Ceph management and monitoring application through which you can inspect and administer various aspects and resources within the cluster.
+
+## Problem description
+
+While it's very likely that the Ceph dashboard meets some accessibility guidelines (ARIA attributes, alt text, etc), we currently ignore its level of conformance to WCAG 2.1 standard (levels A, AA and AAA).
+
+In a recent article published by the world bank on [disability inclusion](https://www.worldbank.org/en/topic/disability#1), disabilities affect about 1
+billion people world wide. These people use web technologies in a variety of ways. Some
+primarily navigate sites by means of a keyboard alone while others with vision disabilities rely
+on assistive technologies like screen readers.
+
+By ensuring the dashboard meets WCAG 2.1 level A compliance, blind and visually impaired ceph users will not be left out and will be able to manage their ceph clusters comfortably.
+
+## Project
+
+For the past few months we have been working on updating the dashboard to meet WCAG 2.1 level A conformance so that blind and visually impaired ceph users can also use it comfortably to manage and monitor their ceph clusters.
+
+To achieve this goal, we had to:
+
+- Test and identify areas of the dashboard in need of an accessibility update
+- Apply fixes based on WCAG 2.1 guidelines
+- Add automated testing to prevent regressions
+
+### Identifying accessibility issues
+
+After testing several recommended [accessibilty checkers](https://www.w3.org/WAI/ER/tools/?q=wcag-21-w3c-web-content-accessibility-guidelines-21) by the W3C, we chose the [accessibility insights plugin](https://accessibilityinsights.io/docs/en/web/overview/) to help quickly identify the most common accessibility issues for each page of the the dashboard.
+
+### Accessibility improvements
+
+Based on the identified issues we we improved the accessibility of several key components including:
+
+#### Login page
+
+By improving the color contrast of the login form input fields users can now clearly read the labels and use the keyboard to navigate the inputs
+
+![login page](./images/login.png)
+
+#### landing page
+
+The dashboard page displays key monitoring information about cluster status, capacity and performance. We improved on the following aspects:
+
+- updated the cluster labels (HEALTH_WARN, HEALTH_OK, HEALTH_ERROR) to be more descriptive (Warning, Ok, Error)
+
+- Used darker colors for labels to make them more legible
+- Added accessible names to informative icons for screen readers
+
+![Dashboard page](./images/dashboard.png)
+
+#### Navigation
+
+The navigation component enables quick access to all the dashboard's sections. We were able to make the following accessibility updates.
```

Comment:
I'd use first-person language throughout your blog post (*I* did X instead of *we* did X). This was your project-- don't minimize your hand in everything!

```suggestion
The navigation component enables quick access to all the Dashboard's sections. I was able to make the following accessibility updates:
```